Source/ImPlotWidgets/Public/LineFilledGraphWidget.h

```diff
@@ -16,7 +16,8 @@ class IMPLOTWIDGETS_API SFilledGraphWidget : public SCompoundWidget
 public:
 	SLATE_BEGIN_ARGS(SFilledGraphWidget)
 	{}
-	SLATE_ARGUMENT(TArray<FDataPoint>, InDataArray2D)
+	SLATE_ARGUMENT(TArray<TArray<FDataPoint>>, DataArrays)
```

Comment:
From a _user's_ perspective here I think it'd be better to make a struct that contained all the info for one data set and have an array of those. We'd also need to think about what data structure will require the least amount of reprocessing by the user and by this widget. This is probably something to have a conversation about rather than something to change for this review though.

```
struct F2DPlotData
{
    TArray<FDataPoint> Data;
    FText SeriesName;
    //Future stuff?
    //FColor LineColor;
    //etc
};

Reply:
Made a task to do this in the future.